olive/engine/engine.py

```diff
@@ -1031,6 +1027,9 @@ def _evaluate_model(
         )
         return signal
 
+    def _get_prefix_output_name(self, output_name: str, accelerator_spec: "AcceleratorSpec"):
```

Comment:
>_get_prefix_output_name [](http://example.com/codeflow?start=8&length=23)

seems it can be staticmethod since self is not used?

Reply:
updated.